bin/gtf_to_table.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Modified from https://www.biostars.org/p/140471/#140798
+
+# Check for correct number of arguments
+if [ "$#" -ne 3 ]; then
+  echo "Usage: $0 <input_file> <output_file> <gtf_source>"
+  exit 1
+fi
+
+# Input and output file names
+file="$1"
+out="$2"
+gtf_source="$3"
+
+# Perform the zcat, awk, and sed commands based on annotation gtf source, and save the output to the specified output file
```

Comment:
I just have to trust this works, there is no way I am going to review this script.

Reply:
It will probably not work for super old gtf files as this function requires a specific input format. But I don't see why anyone would use outdated references for single cell data that have only recently been available.
I can test until what version backwards it breaks for gencode/ensembl and we can put a note or something

Reply:
definitely no need to support ancient versions.